{"uuid": "76ccaf16-0815-4efe-a8a4-946109451bb5", "counter": 40, "stdout": "ERROR! We were unable to read either as JSON nor YAML, these are the errors we got from each:\r\nJSON: Expecting value: line 1 column 1 (char 0)\r\n\r\nSyntax Error while loading YAML.\r\n  did not find expected key. while parsing a block mapping\r\n  in \"<unicode string>\", line 14, column 3\r\ndid not find expected key\r\n  in \"<unicode string>\", line 17, column 24\r\n\r\nThe error appears to be in '/home/bakvivas/Documents/Projects/ansible-role-3x-ui/roles/3x_ui_container/tasks/start_container.yml': line 17, column 24, but may\r\nbe elsewhere in the file depending on the exact syntax problem.\r\n\r\nThe offending line appears to be:\r\n\r\n    msg: \"No variable vpn_username need\"\r\n  when: 'vpn_username' in extravars.keys()\r\n                       ^ here\r\nThis one looks easy to fix. It seems that there is a value started\r\nwith a quote, and the YAML parser is expecting to see the line ended\r\nwith the same kind of quote. For instance:\r\n\r\n    when: \"ok\" in result.stdout\r\n\r\nCould be written as:\r\n\r\n   when: '\"ok\" in result.stdout'\r\n\r\nOr equivalently:\r\n\r\n   when: \"'ok' in result.stdout\"", "start_line": 44, "end_line": 74, "runner_ident": "80187ff0-ff00-4940-8e32-1950dd897ca8", "event": "error", "pid": 10003, "created": "2024-12-03T22:41:32.065822+00:00", "parent_uuid": "985f4199-2370-58df-eb32-000000000012", "event_data": {"playbook": "playbook.yml", "playbook_uuid": "0dcd79ae-7c09-4082-86bd-a513c1ca34d0", "play": "start docker", "play_uuid": "985f4199-2370-58df-eb32-000000000002", "play_pattern": "3x_ui_server", "task": "Start container", "task_uuid": "985f4199-2370-58df-eb32-000000000012", "task_action": "include_tasks", "resolved_action": null, "task_args": "", "task_path": "/home/bakvivas/Documents/Projects/ansible-role-3x-ui/roles/3x_ui_container/tasks/main.yml:26", "role": "3x_ui_container", "error": true, "uuid": "76ccaf16-0815-4efe-a8a4-946109451bb5"}}